lms/djangoapps/courseware/views/views.py

```diff
@@ -629,6 +629,11 @@ def create_page_context(self, request, course=None, tab=None, **kwargs):
             'uses_pattern_library': not uses_bootstrap,
             'disable_courseware_js': True,
         }
+
+        # Avoid Multiple Mathjax loading on the 'user_profile'
+        if 'profile_page_context' in kwargs:
```

Comment:
In the given scenario, we are only updating the context passed to the **tab-view.html** file with load_mathjax flag. The **profile_page_context** is used by the _discussion_profile_page.html_ to generate the page as it contains the information related to that page. Passing all that information to the context of the tab-view is unnecessary. We only want to pass the 'load_mathjax' flag so that tab-view doesn't load the mathjax on the **user_profile** discussion page.